fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -1152,6 +1165,38 @@ private void checkFunction(String fnName, FunctionCallExpr node, Type[] argument
                     }
                     break;
                 }
+                case FunctionSet.NAME_STRUCT: {
+                    if (node.getChildren().size() < 2) {
+                        throw new SemanticException(fnName + " should have at least two inputs", node.getPos());
```

Comment:
are there no empty struct?

Reply:
I think we don't need empty struct? it's meaningless

Reply:
need be align to other systems, like trino

Reply:
trino doesn't support empty paramter, empty struct isn't empty map, emptry struct has none any subfield, why need empty struct if you don't need any subfield?